test/k8sT/Policies.go

```diff
@@ -1581,27 +1582,18 @@ var _ = Describe("K8sPolicyTest", func() {
 		}
 
 		testConnectivitytoRedis := func() {
-			webPods, err := kubectl.GetPodsNodes(helpers.DefaultNamespace, "-l k8s-app.guestbook=web")
-			Expect(err).To(BeNil(), "Cannot get web pods")
-
-			serviceIP, port, err := kubectl.GetServiceHostPort(helpers.DefaultNamespace, "redis-master")
-			Expect(err).To(BeNil(), "Cannot get hostPort of redis-master")
-
-			serviceName := "redis-master"
-			err = kubectl.WaitForKubeDNSEntry(serviceName, helpers.DefaultNamespace)
-			Expect(err).To(BeNil(), "DNS entry is not ready after timeout")
+			webPods, err := kubectl.GetPodsNodes(helpers.DefaultNamespace, "-l app=guestbook")
+			Expect(err).To(BeNil(), "Error retrieving web pods")
+			Expect(webPods).ShouldNot(BeEmpty(), "Cannot retrieve web pods")
```

Comment:
These `Expect`s should be `ExpectWithOffset` because we're inside a function, rather than inside an `It()` directly.

Reply:
Fixed. Then, if I'm understanding this correctly, we actually have incorrect `Expect` calls all over our tests, right?

Reply:
Yes , unfortunately we do :(